<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fightwindow</class>
 <widget class="QWidget" name="fightwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Plants vs Zombies</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/resource/newwindowicon.png</normaloff>:/resource/newwindowicon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QWidget" name="page">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>522</width>
         <height>87</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>522</width>
         <height>87</height>
        </size>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/resource/ChooserBackground.png);</string>
       </property>
       <widget class="QPushButton" name="pushButton_card_sunflower">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>10</y>
          <width>50</width>
          <height>70</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/resource/向日葵卡片.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_card_pea">
        <property name="geometry">
         <rect>
          <x>135</x>
          <y>10</y>
          <width>50</width>
          <height>70</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/resource/豌豆卡片.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_card_nut">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>10</y>
          <width>50</width>
          <height>70</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/resource/土豆卡片.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_plate">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>CrossCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/resource/plate.png);
color: rgb(7, 7, 7);
font: 20pt &quot;AcadEref&quot;;</string>
        </property>
        <property name="text">
         <string>50</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>800</width>
         <height>600</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: none;
border-image: none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="fightwindow_resource.qrc">:/resource/fightwindow_background.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
      <zorder>label</zorder>
      <zorder>widget</zorder>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
  <include location="fightwindow_resource.qrc"/>
 </resources>
 <connections/>
</ui>
